### ISSUE TITLE
Cloning of repository failed

### ISSUE THREAD

**Issue by frank-steimke** [transpect/docx2jats-demo#1]
Hi,

following your tutorial at http://transpect.github.io/tutorial.html i'd like to clone the repository. It fails with this error message:

```
Cloning into 'C:/kosit/docx2jats-demo/schema/hub/css'...
fatal: unable to connect to github.com:
github.com[0: 140.82.121.3]: errno=Unknown error

fatal: clone of 'git://github.com/le-tex/CSSa' into submodule path 'C:/kosit/docx2jats-demo/schema/hub/css' failed
```

The whole process ends with: Failed to recurse into submodule path 'schema/hub'

Happens with git on windows. We are behind a firewall, but other submodules are checked out without any problems.

Thanks,
Frank Steimke, Bremen

**Comment by gimsieke** [comment]
Yes, I noticed the message "The unauthenticated git protocol on port 9418 is no longer supported." when cloning with `git clone https://github.com/transpect/docx2jats-demo.git --recursive`. There may be even more issues since the project hasn’t been updated for some time now. We will try to amend the issue quite soon, but I cannot promise an exact date. Thanks for reporting it, and thanks for your interest in the tool.

**Comment by mladenbabic** [comment]
Hi, I've also experienced a problem with this repository. I followed everything and if I start conversion with `calabash/calabash.sh a9s/common/xpl/docx2jats.xpl file=docx/CSMI_2013_601.docx` 
I got this:
`Error: Unable to initialize main class com.xmlcalabash.drivers.Main
Caused by: java.lang.NoClassDefFoundError: net/sf/saxon/s9api/SaxonApiException`

Any idea what could be wrong? The documentation also confusing in some cases. Please, try to update the documentation, because I really want to use this project and I have found it very helpful. 

Tnx in advance.
Mladen

**Comment by gimsieke** [comment]
A fix is imminent.

**Comment by gimsieke** [comment]
(still working on it, but I’m optimistic that it will be fixed by 18:00 CEST today)

**Comment by gimsieke** [comment]
It should be running again if you clone it afresh. We’ll need to update some modules and check whether the output is as expected in the near future (over the weekend, I hope)

**Comment by mladenbabic** [comment]
Thanks a lot. I'm going to check it now.

**Comment by mladenbabic** [comment]
Checked and it works great! Thanks once again.
